thymeleaf-spring6/BUILD.txt

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
As with the PR for the `thymeleaf` code module, please note that Thymeleaf now is in `3.0.14.RELEASE` and there is a new `3.1-dev` branch which this should be rebased on.

Also note that 3.0.14.RELEASE includes some changes related to restricted mode detection checks that might have made these preliminary classes in the new `thymeleaf-spring6` module outdated.